components/collection/EditModal.vue

```diff
@@ -186,10 +280,17 @@ const disabled = computed(() => {
   const descriptionChanged = props.collection.description !== description.value
   const hasBannerChanged = (!bannerUrl.value && Boolean(props.collection.banner)) || Boolean(banner.value)
   const hasMaxChanged = max.value !== props.collection.max
+  const holderOfCollectionIdChanged = holderOfCollectionId.value !== originalHolderOfCollectionId.value
+  const invalidPublicCollection = selectedMintingType.value === 'Public' && !mintingPrice.value
+
+  const invalidHolderOfCollection = selectedMintingType.value === 'HolderOf' && !holderOfCollectionId.value
```

Comment:
Create a computed , you can reuse it inside the template 

```suggestion
  const invalidHolderOfCollection = isHolderOfMintingType.value && !holderOfCollectionId.value
```

Reply:
updated